webapp/components/wpt-results.html

```diff
@@ -776,10 +774,12 @@ <h3>
         this.navigateToPath(e.detail.path);
       }
 
-      handleSearchCommit() {
+      handleSearchCommit(e) {
+        const detail = e.detail;
         // Fetch search results when test-search signals that user has committed
         // to search string (by pressing <Enter>).
-        this.fetchResults();
+        this.fetchResults(this.structuredQueries ?
```

Comment:
Nit: wrap like

        this.fetchResults(this.structuredQueries
          ? detail.structuredQuery
          : detail.query);